packages/access-api/src/types/delegations.ts

```diff
@@ -0,0 +1,28 @@
+import * as Ucanto from '@ucanto/interface'
+
+export type Resolvable<T> = T | Promise<T>
+
+export interface DelegationsStorage<
+  Cap extends Ucanto.Capability = Ucanto.Capability
+> {
+  /**
+   * push items into storage
+   *
+   * @param delegations - delegations to store
+   */
+  push: (
```

Comment:
I'm not convinced this is a `push` - it's not ordered in any way - `put` may be more appropriate or `putMany`

Reply:
renamed to `putMany` https://github.com/web3-storage/w3protocol/pull/427/commits/022dc2167647dcbabb35b6ab0450dafd9c708a84#diff-0c692d0f8036f645c9fe7c647d216265dedfb7a84decb52490918e43750a9e9eR11